<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Objets présent dans la scène</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>641</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listeFormes"/>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>431</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cercle</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="couleurCercle">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>Noir</string>
           </property>
           <item>
            <property name="text">
             <string>Noir</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blanc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Jaune</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rouge</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vert</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bleu</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Centre:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>x: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="centreX">
             <property name="toolTip">
              <string>coordonnées en x du point</string>
             </property>
             <property name="minimum">
              <double>-990.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>y: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="centreY">
             <property name="toolTip">
              <string>coordonnées en y du point</string>
             </property>
             <property name="minimum">
              <double>-990.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Rayon: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="rayon">
             <property name="maximum">
              <double>199.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>5.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ajout_cercle">
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Segment</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QComboBox" name="couleurSegment">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>Noir</string>
           </property>
           <item>
            <property name="text">
             <string>Noir</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blanc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Jaune</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rouge</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vert</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bleu</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>p1: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>x: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sP1X">
             <property name="toolTip">
              <string>coordonnées en x du point</string>
             </property>
             <property name="minimum">
              <double>-990.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>y: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sP1Y">
             <property name="toolTip">
              <string>coordonnées en y du point</string>
             </property>
             <property name="minimum">
              <double>-990.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>p2: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>x: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sP2X">
             <property name="toolTip">
              <string>coordonnées en x du point</string>
             </property>
             <property name="minimum">
              <double>-990.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>y: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sP2Y">
             <property name="toolTip">
              <string>coordonnées en y du point</string>
             </property>
             <property name="minimum">
              <double>-990.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ajout_segment">
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Triangle</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>446</width>
        <height>196</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QComboBox" name="couleurTriangle">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>Noir</string>
           </property>
           <item>
            <property name="text">
             <string>Noir</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blanc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Jaune</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rouge</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vert</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bleu</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>p1: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>x: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="tP1X">
             <property name="toolTip">
              <string>coordonnées en x du point</string>
             </property>
             <property name="minimum">
              <double>-990.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>y: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="tP1Y">
             <property name="toolTip">
              <string>coordonnées en y du point</string>
             </property>
             <property name="minimum">
              <double>-990.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>p2: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>x: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="tP2X">
             <property name="toolTip">
              <string>coordonnées en x du point</string>
             </property>
             <property name="minimum">
              <double>-990.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>y: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="tP2Y">
             <property name="toolTip">
              <string>coordonnées en y du point</string>
             </property>
             <property name="minimum">
              <double>-990.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>p3: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>x: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="tP3X">
             <property name="toolTip">
              <string>coordonnées en x du point</string>
             </property>
             <property name="minimum">
              <double>-990.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>y: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="tP3Y">
             <property name="toolTip">
              <string>coordonnées en y du point</string>
             </property>
             <property name="minimum">
              <double>-990.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
             <property name="singleStep">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ajout_triangle">
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Polygone</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Groupe</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>570</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer la forme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dessiner_forme">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>570</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Dessiner la forme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dessiner_tout">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>570</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Tout dessiner</string>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>tabWidget</zorder>
   <zorder>horizontalLayoutWidget_7</zorder>
   <zorder>supprimer</zorder>
   <zorder>dessiner_forme</zorder>
   <zorder>dessiner_tout</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
